modules/images/dominant-color/load.php

```diff
@@ -257,41 +258,11 @@ function dominant_color_set_image_editors() {
  * @since n.e.x.t
  *
  * @param int $attachment_id The attachment ID.
- * @return string|WP_Error The dominant color of the image, or WP_Error on error.
- */
-function dominant_color_get_dominant_color( $attachment_id ) {
-	$file = wp_get_attachment_file_path( $attachment_id );
-	if ( ! $file ) {
-		$file = get_attached_file( $attachment_id );
-	}
-	add_filter( 'wp_image_editors', 'dominant_color_set_image_editors' );
-	$editor = wp_get_image_editor( $file );
-	remove_filter( 'wp_image_editors', 'dominant_color_set_image_editors' );
-
-	if ( is_wp_error( $editor ) ) {
-		return $editor;
-	}
-	if ( ! method_exists( $editor, 'get_dominant_color' ) ) {
-		return new WP_Error( 'unable_to_find_method', __( 'Unable to find get_dominant_color method', 'performance-lab' ) );
-	}
-
-	$dominant_color = $editor->get_dominant_color();
-	if ( is_wp_error( $dominant_color ) ) {
-		return $dominant_color;
-	}
-
-	return $dominant_color;
-}
-
-/**
- * Computes whether the given attachment image has transparency.
+ * @return array|WP_Error The dominant color of the image, or WP_Error on error.
  *
- * @since n.e.x.t
- *
- * @param int $attachment_id The attachment ID.
- * @return bool|WP_Error True if the color has transparency or WP_Error on error.
+ * @access private
```

Comment:
This should go right under the `@since` line.

Also, can you please update the function description accordingly? It should now say that it computes _both_ the dominant color and whether the image has transparency.